lib/Report/Renderer/ConsoleRenderer.php

```diff
@@ -47,55 +37,14 @@ public function __construct(OutputInterface $output, Printer $printer)
      */
     public function render(Reports $reports, Config $config): void
     {
-        foreach ($reports as $report) {
-            if ($title = $report->title()) {
-                $this->output->writeln(sprintf('<title>%s</title>', $title));
-                $this->output->writeln(sprintf('<title>%s</title>', str_repeat('=', strlen($title))));
-                $this->output->write(PHP_EOL);
-            }
-
-            if ($description = $report->description()) {
-                $this->output->writeln(sprintf('<title>%s</title>', $title));
-                $this->output->writeln(sprintf('<description>%s</description>', $description));
-                $this->output->writeln('');
-            }
-
-            foreach ($report->tables() as $table) {
-                $this->output->writeln(sprintf('%s', $table->title()));
-                $this->renderTableElement($table, $config);
-            }
-        }
-    }
-
-    protected function renderTableElement(PhpBenchTable $table, Config $config): void
-    {
-        $rows = [];
-
-        $consoleTable = new Table($this->output);
-        $consoleTable->setStyle($config['table_style']);
-        $consoleTable->setHeaders($table->columnNames());
-        $consoleTable->setRows($this->buildRows($table));
-        $consoleTable->render();
-        $this->output->writeln('');
+        $this->renderer->render($reports);
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
      */
     public function configure(OptionsResolver $options): void
     {
-        $options->setDefaults([
-            'table_style' => 'default',
-        ]);
-        $options->setAllowedTypes('table_style', ['string']);
-    }
-
-    private function buildRows(PhpBenchTable $table): array
-    {
-        return array_map(function (TableRow $row) {
-            return array_map(function (Node $node) {
-                return $this->printer->print($node);
-            }, $row->cells());
-        }, $table->rows());
+        $options->setDefault('table_style', null);
```

Comment:
This option is not propagated to the indivdual renderers. It should be marked as dead and the responsiblity moved to the table generator (??).